docker/crypto-detector/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN pip install -r requirements.txt
 
 COPY crypto-detector.py /app
 
-CMD ["python3", "/app/crypto-detector.py", "-b", "kafka-service:9092", "-c", "CDS", "-p", "crypto-result"]
+ENTRYPOINT ["python3", "/app/crypto-detector.py", "-b", "kafka:9092", "-c", "input", "-p", "output"]
```

Comment:
Why do you use `ENTRYPOINT` to run the script with predefined values for the input arguments? I understand you want to provide flexibility with `ENTRYPOINT`, but the input arguments should not be set here. The code should be something like:

```ENTRYPOINT ["python3", "/app/crypto-detector.py"]```

Then if you want to have default values in case no input arguments are provided in the execution of the container, these default values should be defined within the Python script.